src/Tasks/Microsoft.NET.Build.Tasks/GenerateBundle.cs

```diff
@@ -0,0 +1,37 @@
+﻿// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using Microsoft.Build.Framework;
+using Microsoft.NET.HostModel.Bundle;
+using System.Collections.Generic;
+
+namespace Microsoft.NET.Build.Tasks
+{
+    public class GenerateBundle : TaskBase
+    {
+        [Required]
+        public ITaskItem[] FilesToBundle { get; set; }
+        [Required]
+        public string AppHostName { get; set; }
+        [Required]
+        public bool EmbedPDBs { get; set; }
```

Comment:
```suggestion
        public bool IncludeSymbols { get; set; }
```

I think it is worth using the same terminology throughout, and the public facing name is symbols for this feature.